.circleci/config.yml

```
@@ -21,12 +21,17 @@ jobs:
           name: gen-coverage
           command: |
             make coverage
-            coveralls
+            if [ $COVERALLS_REPO_TOKEN ]; then
+              coveralls
+            fi
 
       - run:
           name: upload-coverage
           command: |
-            bash <(curl -s https://codecov.io/bash)
+            if [ $COVERALLS_REPO_TOKEN ]; then
+              bash <(curl -s https://codecov.io/bash)
+            fi
+
```

```suggestion
            if [ $COVERALLS_REPO_TOKEN ]; then
              coveralls
            fi
```

issue: We don't use `codecov` anymore.

We can replace this step to use `coveralls` since we don't use the `codecov` solution anymore for our projects.